package.json

```diff
@@ -21,6 +21,7 @@
     "@nivo/pie": "^0.79.1",
     "@nivo/stream": "^0.79.1",
     "@types/react-table": "^7.7.8",
+    "call-once": "^1.0.3",
```

Comment:
What do these three packages do?

If they are for tests they can be placed in devDependencies.

Reply:
Was a package for the trying to fix the unit tests. I have since removed it in the latest version, but the copy link button tests are failing :/